validator/main.go

```diff
@@ -318,6 +320,27 @@ func main() {
 			Destination: &disableDevCharSymlinkCreation,
 			EnvVars:     []string{"DISABLE_DEV_CHAR_SYMLINK_CREATION"},
 		},
+		&cli.BoolFlag{
+			Name:        "driver-managed-by-gpu-operator",
+			Value:       true,
+			Usage:       "indicates if the NVIDIA GPU driver is managed by GPU Operator",
+			Destination: &driverManagedByOperator,
+			EnvVars:     []string{"DRIVER_MANAGED_BY_GPU_OPERATOR"},
+		},
+		&cli.StringFlag{
+			Name:        "host-root",
+			Value:       "/",
+			Usage:       "root path of the underlying host",
+			Destination: &hostRootFlag,
+			EnvVars:     []string{"HOST_ROOT"},
+		},
+		&cli.StringFlag{
+			Name:        "driver-root",
+			Value:       defaultDriverRoot,
+			Usage:       "root path of a containerized NVIDIA driver installation",
```

Comment:
Is this the path on the host or in the container?

Reply:
As currently implemented, it happens to be both the path on the host AND in the driver-validation container, due to the way the volumeMount is configured for the daemonset.

It may be good to split this into `driver-root` and `driver-root-ctr-path` fields for clarity (even if they are the same values to begin with).

Reply:
For clarity, I have updated the driver validator to have two flags -- `driver-install-dir` and `driver-install-dir-ctr-path`.